.clang-format

```diff
@@ -0,0 +1,17 @@
+# A clang-format style that approximates Python's PEP 7
```

Comment:
This whole file seems like something that should not go in as part of a feature branch.  If it goes in at all (and maybe it should!) it should go in on its own, first, and then be part of your baseline.

Reply:
This PR adds the only C to this project. It doesn't make much sense to add a C formatting tool to a baseline repo that has no C code in it, since the test of this file is its performance on the C files.